index.html

```diff
@@ -3257,15 +3257,14 @@ <h2>Syntaxes</h2>
 <a href="#advanced-concepts"></a> is the canonical structural representation of
 a <a>verifiable credential</a> or <a>verifiable presentation</a>. All
 serializations are representations of that data model in a specific format. This
-section specifies how the data model is realized in JSON-LD and plain JSON.
-Although syntactic mappings are provided for only these two syntaxes,
+section specifies how the data model is realized in JSON-LD for the base
+media type (`application/vc+ld+json`).
+Although syntactic mappings are provided for JSON-LD,
 applications and services can use any other data representation syntax (such as
-XML, YAML, or CBOR) that is capable of expressing the data model. As the
-<a>verification</a> and <a>validation</a> requirements are defined in terms of
-the data model, all serialization syntaxes have to be deterministically
+XML, YAML, or CBOR) that is capable of being mapped back to the base media type.
```

Comment:
```suggestion
XML, YAML, or CBOR) that is capable of being mapped back to `application/vc+ld+json`.
```

Reply:
I am concerned about the lack of "losslessly" in these paragraphs, as that has been a key factor in our discussions.